samcli/local/docker/lambda_container.py

```diff
@@ -293,13 +294,21 @@ def _get_entry_point(runtime, debug_options=None):
                        "/var/runtime/awslambda/bootstrap.py"
                    ]
 
+        elif runtime == Runtime.python37.value:
```

Comment:
at some point we need to clean up with the if else and make it cleaner. Maybe something like a dictionary key dispatch would work here.

https://github.com/awslabs/aws-sam-cli/issues/853